src/screens/Followings/elements/TopicScreen/SuggestionTopic.js

```diff
@@ -1,21 +1,21 @@
 import React, {memo} from 'react';
 import {FlatList, ScrollView, StyleSheet, Text, TouchableOpacity, View} from 'react-native';
 
-import {colors} from '../../../../utils/colors';
 import {fonts} from '../../../../utils/fonts';
+import {COLORS} from '../../../../utils/theme';
 
 const styles = StyleSheet.create({
   titleText: {
     fontFamily: fonts.inter[500],
     fontWeight: 'bold',
     fontSize: 16,
-    color: colors.black
+    color: COLORS.black
   },
   suggestionContainer: {
     paddingHorizontal: 20
   },
   capsulContainer: {
-    backgroundColor: colors.alto,
+    backgroundColor: COLORS.alto,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is consistent with the previous ones, replacing hardcoded color values with references to color constants from a theme file.

```diff
- backgroundColor: colors.alto,
+ backgroundColor: COLORS.alto,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->